src/frontend/src/flows/about/index.ts

```diff
@@ -1,10 +1,12 @@
 import { html, render } from "lit-html";
 import { hydrate } from "lit-html/experimental-hydrate.js";
 import { compatibilityData } from "../../components/compatibilityChart";
+import { mainWindow } from "../../components/mainWindow";
 
 // The About page
-export const pageContent = html`
-  <div class="l-container l-container--wide c-card c-card--highlight">
+export const pageContent = mainWindow({
```

Comment:
Wrapping all the functions (here and elsewhere) with `mainWindow` breaks git and makes the diff hard to read and look huge. How about something like this?

```
export const pageContent = mainWindow({ isWideContainer: true, slot: innerPageContent });

const innerPageContent = /* same as before */;
```

It would also clarify what is specific to this page and what isn't